docs/src/how-to/install/infrastructure-configuration.md

```diff
@@ -587,13 +587,12 @@ team-settings:
     IS_SELF_HOSTED: "true"
 ```
 
-Second, also set the option under the `account-pages` section:
+Second, also set the option for `account-pages` helm chart:
 
 ```yaml
-# NOTE: Only relevant if you want account-pages
 account-pages:
-  envVars:
-    IS_SELF_HOSTED: "true"
+envVars:
```

Comment:
Hm, can we change the indentation here? Doesn't this change the semantics/meaning of this YAML snippet?